policyengine_us/parameters/gov/states/ma/tax/income/deductions/rent/cap.yaml

```diff
@@ -1,14 +1,14 @@
 description: Maximum rental deduction for MA income taxes.
 SINGLE:
-  2011-01-01: 3_000
+  2011-01-01: 4_000
```

Comment:
the increase happened in 2023, we leave the original value and add a value starting in 2023

Need new 2023 tests (the existing ones remain untouched)